From 22b005a59ff3db88783cd1f329c97b823ea3f03f Mon Sep 17 00:00:00 2001
From: Andrew Bartlett <abartlet@samba.org>
Date: Sun, 16 Jun 2013 13:32:10 +1000
Subject: [PATCH] heimdal: Remove samba4kgetcred again

This tool is not used in our tests, is not installed, and causes the build
to fail on current debian unstable due to a new binutils (when linked
with the system Heimdal).

Andrew Bartlett
---
 source4/heimdal_build/wscript_build | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/source4/heimdal_build/wscript_build b/source4/heimdal_build/wscript_build
index 44634b4..365d319 100644
--- a/source4/heimdal_build/wscript_build
+++ b/source4/heimdal_build/wscript_build
@@ -946,12 +946,6 @@ HEIMDAL_BINARY('samba4kinit',
     install=False
     )
 
-HEIMDAL_BINARY('samba4kgetcred',
-    'kuser/kgetcred.c',
-    deps='krb5 heimntlm roken HEIMDAL_VERS hcrypto',
-    install=False
-    )
-
 HEIMDAL_BINARY('samba4kpasswd',
     'kpasswd/kpasswd.c',
     deps='krb5 heimntlm roken HEIMDAL_VERS hcrypto',
-- 
1.7.11.7

